ros/src/tl_detector/light_classification/ssd_tl_classifier.py

```diff
@@ -5,25 +5,38 @@
 import tensorflow as tf
 import numpy as np
 import cv2
+from PIL import Image, ImageFont, ImageDraw
```

Comment:
Please, move this line near to other lines that start with `from ...`. Just for consistency.